scripts/build-and-pack.sh

```diff
@@ -41,4 +41,19 @@ echo "npm pack in $PACKAGE_DIR"
 npm pack
 
 echo "copy $PACKAGE package to $PROJECT_BASE_DIR"
-cp "$PACKAGE_DIR"/*.tgz "$PROJECT_BASE_DIR"
\ No newline at end of file
+cp "$PACKAGE_DIR"/*.tgz "$PROJECT_BASE_DIR"
```

Comment:
Should this .sh script lint test as well?

Reply:
I don't think it is necessary. Since our github ci pipeline will guard us.